core/src/main/java/nl/nn/adapterframework/webcontrol/action/BrowseJdbcTable.java

```diff
@@ -59,47 +60,22 @@ public ActionForward executeSub(
 		}
 
 		IniDynaActionForm browseJdbcTableForm = (IniDynaActionForm) form;
-		Cookie[] cookies = request.getCookies();
-
-		if (null != cookies) {
-			for (int i = 0; i < cookies.length; i++) {
-				Cookie aCookie = cookies[i];
-
-				if (aCookie
-					.getName()
-					.equals(
-						AppConstants.getInstance().getProperty(
-							"WEB_JDBCBROWSECOOKIE_NAME"))) {
-					StringTagger cs = new StringTagger(aCookie.getValue());
-
-					log.debug("restoring values from cookie: " + cs.toString());
-					try {
-						browseJdbcTableForm.set(
-							"jmsRealm",
-							cs.Value("jmsRealm"));
-						browseJdbcTableForm.set(
-							"tableName",
-							cs.Value("tableName"));
-						browseJdbcTableForm.set(
-							"where",
-							cs.Value("where"));
-						browseJdbcTableForm.set(
-							"order",
-						cs.Value("order"));
-						browseJdbcTableForm.set(
-							"numberOfRowsOnly",
-							new Boolean(cs.Value("numberOfRowsOnly")));
-						browseJdbcTableForm.set(
-							"rownumMin",
-							new Integer(cs.Value("rownumMin")));
-						browseJdbcTableForm.set(
-							"rownumMax",
-							new Integer(cs.Value("rownumMax")));
-					} catch (Exception e) {
-						log.warn("could not restore Cookie value's", e);
-					}
-				}
 
+		Cookie cookie = CookieUtil.getCookie(request, AppConstants.getInstance().getProperty("WEB_JDBCBROWSECOOKIE_NAME"));
+		if (null != cookie) {
+			StringTagger cs = new StringTagger(cookie.getValue());
+
+			log.debug("restoring values from cookie: " + cs.toString());
+			try {
+				browseJdbcTableForm.set("jmsRealm", cs.Value("jmsRealm"));
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.